addons/hr_expense/views/res_config_settings_views.xml

```diff
@@ -43,6 +43,10 @@
                                      string="Employee Expense Journal">
                                 <field name="expense_journal_id"/>
                             </setting>
+                            <setting company_dependent="1" help="Default outstanding account for expenses paid by employees."
+                                     string="Employee Expense Oustanding Account">
+                                <field name="expense_outstanding_account_id"/>
```

Comment:
tbh, it feels like unfinished business: we removed the outstanding account but we still need it for expenses... :see_no_evil: and we won't have it automatically filled with the CoA generation :shrug:

Reply:
It was discussed with TSB, there is no other choice because we always need to generate an entry for company expenses, but we don't want to depend on the configuration of the payment method